plugins/hydra_ax_sweeper/news/_template.rst

```diff
@@ -0,0 +1,35 @@
+{% for section in sections %}
```

Comment:
Sorry for the noob question. would it be possible to share the `_template.rst` some way?

Reply:
Good question.
Looking at the code of towncrier, it looks like it's possible to specify the template file to use in pyproject.toml.